linode/manager
# fix: [M3-7565] - Fix Radio size prop not affecting the radio button's dimensions

## Description 📝
We realized this issue while working on the Radio storybook: [changing the size prop of the radio did not actually affect the radio's appearance](https://github.com/linode/manager/pull/9994#issuecomment-1858122160).

I noticed the size of the radio buttons was always 25px - which was coming from the width and height set in `radio.svg` and `radioRadioed.svg`. The font-size being set had no effect on the radio's dimensions.


## Changes  🔄
- Passed `height` and `width` to `RadioIconRadioed` and `RadioIcon` based on the size prop's value
- Updated expected sizes in pixels per @gitkcosby's suggestion - 16px for small and 20px for medium.

## Target release date 🗓️
NA

## Preview 📷


## How to test 🧪


### Reproduction steps
- Set the size prop to "small" and then "medium" of any radio button. Inspect on the browser - you should see 25px for both cases

### Verification steps
- Pull the PR, Set the size prop to "small" and then "medium" of any radio button. Inspect on the browser. Small radios should have a width of 16px and medium 20px.
NOTE: It's possible that updating the medium radio size to 20px could cause alignment issues on pages where radios are used. We'll need to test this

## As an Author I have considered 🤔

*Check all that apply*
